File: sbe/binance/spot_3_2/generated-python/spot_sbe/MatchType.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple
from io import BytesIO

class MatchType:
    """Enum values."""
    AUTO_MATCH = 1
    ONE_PARTY_TRADE_REPORT = 2
    NON_REPRESENTABLE = 254
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1

File: sbe/binance/spot_3_2/generated-php/spot_sbe/MatchType.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class MatchType
{
    public const AUTOMATCH = 1;
    public const ONEPARTYTRADEREPORT = 2;
    public const NONREPRESENTABLE = 254;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}
